src/interiorpointsolver.jl

```diff
@@ -709,9 +690,8 @@ function regular!(ips::AbstractInteriorPointSolver)
         switching_condition = is_switching(varphi_d,ips.alpha,ips.opt.s_phi,ips.opt.delta,2.,ips.opt.s_theta)
         armijo_condition = false
         while true
-            ips.x_trial .= ips.x .+ ips.alpha.*ips.dx
+            apply_step!(ips.x_trial,ips.x,ips.dx,ips.alpha)
```

Comment:
instead of using a new function `apply_step!`, how about using directly a BLAS function? 
```julia
copyto!(ips.x_trial, ips.x)
axpy!(ips.alpha, ips.dx, ips.x_trial)
```

Reply:
Yes, using BLAS makes more sense :+1: